docs/install.rst

```diff
@@ -207,6 +207,13 @@ will be installed.
    be improved or have become out of date, please create an issue on
    `XRTpy's GitHub repository`_. It would really help!
 
+
+.. installation:: sphinx-toolbox
+    :pypi:
+    :anaconda:
+    :conda-channels: conda-forge
+    :github:
```

Comment:
Do we need to tweak these settings:

For GitHub it points to the wrong branch:

``python3 -m pip install git+https://github.com/HinodeXRT/xrtpy@master --user`